### DISCUSSION TITLE
Sole proprietorship Ltd organization

### DISCUSSION THREAD

**Discussion by icupeiro** [community/community#61834] [New to GitHub] [answered]
Hello

During the next month, I will start my journey as a freelancer. I want to create an organization to separate my personal account and repos from my freelancer jobs, such as I will be the worker of that organization and I can invite potential collaborators for specific repositories. 

When creating the team, I get the following question:

This organization belongs to:

- My personal account
- A business or institution

What would be the right choice here? The legal formula I chose to be a freelancer is to create an (sole proprietorship) ltd. instead of be an independent worker due to tax/fiscal reasons, so despite it will be only me I will be working for a company (which is my own).

Thanks in advance!

**Comment by priyankarpal** [comment, marked as answer]
@icupeiro it would be most appropriate to select `"A business or institution"` when creating the team. As you mentioned, you have chosen the legal structure of a sole proprietorship ltd. to operate your freelance business. By doing so, you have effectively created a separate legal entity for your freelancing activities, which is considered a business. Even though you are the sole owner and employee of this business, it is still classified as a separate entity from your personal account.

This will help you keep your freelancer jobs, repositories, and other business-related activities separate from your personal account, and also enable you to invite potential collaborators for specific repositories within the organization.
